core/templates/dev/head/base.html

```diff
@@ -255,10 +255,20 @@
                           <ul ng-if="!windowIsNarrow" class="nav oppia-navbar-tabs">
                             <create-activity-button></create-activity-button>
 
-                            <li class="oppia-clickable-navbar-element pull-right">
-                              <a class="oppia-navbar-tab" href="/donate" translate="I18N_TOPNAV_DONATE">
+                          <ul ng-if="windowIsNarrow" class="nav oppia-navbar-tabs-narrow">
```

Comment:
If you look at the indentation and the structure of the HTML tags, the previous ul element has not been closed yet, so you're inserting this ul element in the middle of an existing one.